documentation-site/static/examples/form-control/validation.js

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {useState} from 'react';
+import {FormControl} from 'baseui/form-control';
+import {StatefulInput, SIZE} from 'baseui/input';
+
+export default () => {
+  const [isError, setError] = useState(false);
+
+  return (
+    <React.Fragment>
+      <FormControl
+        error={isError}
+        label="Input label"
+        caption="Enter a number bigger than 10"
+      >
+        <StatefulInput
+          onChange={event => {
+            if (Number(event.target.value) < 11) {
+              return setError(true);
+            }
+            setError(false);
+          }}
+          size={SIZE.compact}
+        />
+      </FormControl>
+    </React.Fragment>
```

Comment:
I'm not convinced that shuttling error through `FormControl` to child is valuable since, it's just as simple to pass directly to the `Input`. Is there more to this that I'm missing?

```jsx
return (
  <FormControl label="Input label" caption="Enter a number bigger than 10">
    <Input
      onChange={event => {setValue(event.target.value)}}
      value={value}
      size={SIZE.compact}
      error={value && Number(value) < 11}
    />
  </FormControl>
);
```

Reply:
yeah, you are right - I was just playing with the concept

Reply:
@chasestarr updated the diff - as per the latest designs, both the caption and the input turns red in the error state

Reply:
image for context
<img width="829" alt="Screen Shot 2019-05-22 at 3 33 10 PM" src="https://user-images.githubusercontent.com/5317799/58213437-f8143500-7ca6-11e9-8a7b-844327257a49.png">

should it follow a `kind` enum pattern like the input? a boolean can't handle 3 different states